org/cloudfoundry.yml

```diff
@@ -2470,6 +2470,19 @@ orgs:
           multiapps: admin
           multiapps-controller: admin
           multiapps-cli-plugin: admin
+      temp-autoscaler-admins:
```

Comment:
Better: temp-ari-autoscaler-admins
Points directly to the WG and matches the naming pattern of most of the other temp admin teams

Reply:
Thanks, good catch. Copy/pasted from the wrong example. :)